### PR TITLE
Bug 1353400, 	follow-up edits to PR#2570 

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/2570
Satisfies:
https://bugzilla.redhat.com/show_bug.cgi?id=1268973
https://bugzilla.redhat.com/show_bug.cgi?id=1353400

### CODE REVIEW

**Comment by ahardin-rh** [comment]
Preview Build:  http://file.rdu.redhat.com/~ahardin/080316/Bug1353400/install_config/adding_hosts_to_existing_cluster.html

**Comment by ahardin-rh** [comment]
@adellape @smunilla @abutcher Please review  :bow:
 http://file.rdu.redhat.com/~ahardin/080316/Bug1353400/install_config/adding_hosts_to_existing_cluster.html

Travis is complaining with a ruby_dep (1.4.0) error message and I am trying to figure out how to resolve it, but this does not impact my local builds and should not impact content review. :surfer: 

cc @bfallonf @vikram-redhat

**Comment by vikram-redhat** [comment]
Thanks @ahardin-rh . @smunilla  @abutcher - Travis is all cool now so let us know if you are good with these docs.

File: install_config/adding_hosts_to_existing_cluster.adoc [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+====
+----
+[OSEv3:children]
+masters
+nodes
+new_nodes
+----
+====
++
+To add a new master hosts, add `new_master`.
```
Comment (abutcher) [inline install_config/adding_hosts_to_existing_cluster.adoc]: s/master/masters

Hosts added to the `[new_masters]` group should also be added to the `[new_nodes]` group when running the master scale up playbook.

Example
```
[masters]
master[1:2].example.com

[new_masters]
master3.example.com

[nodes]
node[1:3].example.com openshift_node_labels="{'region': 'infra'}"
master[1:2].example.com openshift_schedulable=false

[new_nodes]
master3.example.com openshift_schedulable=false
```

**Comment by abutcher** [comment]
LGTM aside from the master+node comment.

File: install_config/adding_hosts_to_existing_cluster.adoc [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+----
+====
++
+See
+xref:../install_config/install/advanced_install.adoc#advanced-host-variables[Configuring
+Host Variables] for more options.
+
+. Hosts added to the *[new_masters]* section should also be added to the
+ *[new_nodes]* section when running the master *_scaleup.yml_* playbook. For
+ example:
```
Comment (adellape) [inline install_config/adding_hosts_to_existing_cluster.adoc]: If this step gets combined w/ the previous step, suggest wording like the following (to match the above wording "when adding a new node"):

"When adding new masters, hosts added to the [new_masters] section must also be added to the [new_nodes] section with the openshift_schedulable=false variable. This ensures the new master host is part of the OpenShift SDN and that pods are not scheduled for placement on them. For example:"

**Comment by ahardin-rh** [comment]
Thanks @abutcher and @adellape! This is updated. 

Build: http://file.rdu.redhat.com/~ahardin/08052016/Bug1353400/install_config/adding_hosts_to_existing_cluster.html

**Comment by abutcher** [comment]
:+1:

**Comment by abutcher** [comment]
I believe this also closes https://github.com/openshift/openshift-docs/issues/2210

**Comment by adellape** [comment]
:+1: LGTM, merging.

**Comment by adellape** [comment]
[rev_history]
|xref:../install_config/adding_hosts_to_existing_cluster.adoc#install-config-adding-hosts-to-cluster[Adding Hosts to an Existing Cluster]
|New topic. Moves existing content on adding node hosts from the xref:../install_config/install/quick_install.adoc#install-config-install-quick-install[Quick Installation] and xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[Advanced Installation] topics and combines with new content on adding master hosts.
%

**Comment by bfallonf** [comment]
@ahardin-rh Big thanks for sorting this out!

**Pull request merged**